src/applications/mhv-medical-records/components/shared/NewRecordsIndicator.jsx

```diff
@@ -162,9 +162,36 @@ const NewRecordsIndicator = ({
   };
 
   return (
-    <div className="vads-u-margin-y--2  no-print" id="new-records-indicator">
-      {content()}
-    </div>
+    <>
+      <div className="vads-u-margin-y--2 no-print" id="new-records-indicator">
+        {content()}
+      </div>
+      <div className="print-only">
+        {!lastSuccessfulUpdate && (
+          <span>
+            <p>
+              <b>
+                Your records may not be up to date.
+                <br />
+              </b>
+              There’s a problem with our system, and we can’t access the date
+              your records were last updated. We’re sorry. Please check back
+              later for updates. If it still doesn’t work, call us at
+              877-327-0022 (TTY:711). We’re here Monday through Friday, 8:00 a.m
+              to 8:00 p. ET.
+            </p>
+          </span>
+        )}
+        {lastSuccessfulUpdate && (
+          <>
```

Comment:
This outer `<>` is unnecessary since you are only returning one tag in this block already (the `<p>` tag).